backend/src/server.js

```diff
@@ -1,17 +1,18 @@
 require('express-async-errors');
 const logger = require('./winston.js');
 const fs = require('fs');
-const config = require('../../config/config.json');
-const api = require('../../config/api.json');
+const CONFIG = require('../../config/config.json');
+const API = require('../../config/api.json');
 const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path');
+const urlJoin = require('url-join');
 const klad = require('../../domain/src/main');
 const errors = require('../../domain/src/errors');
 const validation = require('../../domain/src/validation');
 let app = express();
 
-if (fs.readdirSync('../../').indexOf('logs') < 0)
+if (!fs.existsSync(__dirname + '/../../logs'))
     fs.mkdirSync('../../logs');
```

Comment:
Why are you creating log directory here instead of logging module (winston.js) ?